code/ZAS/XGM_gases.dm

```diff
@@ -99,23 +101,44 @@
 /datum/gas/volatile_fuel
 	id = GAS_VOLATILE
 	name = "Volatile Fuel"
+	short_name = "Fuel<sub>V</sub>"
 
 	specific_heat = 30
 	molar_mass = 0.163 // @MoMMI#9954 roll 32 405
 
-	flags = XGM_GAS_FUEL | XGM_GAS_LOGGED
+	flags = XGM_GAS_LOGGED
 
 /datum/gas/volatile_fuel/is_human_safe(moles, datum/gas_mixture/mixture)
 	return moles/mixture.total_moles() < 0.01
 
 /datum/gas/oxygen_agent_b
 	id = GAS_OXAGENT
 	name = "Oxygen Agent B"
+	short_name = "Oxy Agent<sub>B</sub>"
 
 	specific_heat = 300
 	molar_mass = 0.300
 
-	flags = XGM_GAS_FUEL | XGM_GAS_OXIDIZER | XGM_GAS_CONTAMINANT | XGM_GAS_LOGGED
+	flags = XGM_GAS_LOGGED
 
 /datum/gas/oxygen_agent_b/is_human_safe(moles, datum/gas_mixture/mixture)
 	return moles/mixture.total_moles() < 0.01
+
+/datum/gas/cryotheum
+	id = GAS_CRYOTHEUM
+	name = "Cryotheum"
+	short_name = "O<sub>2</sub>C"
```

Comment:
o2c? wouldnt that just be co2?

Reply:
It's a wacky space CO2 where the central molecule is an O, not the C, thus resonating and something something space bullshit it cools down.

Reply:
Yeah it's space oxygen, the C stands for cryo. I guess it does sound like C carbon huh.